### PR TITLE
Teams as url parameter

### PR DESCRIPTION
**Pull request by CanardSauvage**

### CODE REVIEW

File: dash-core/src/main/resources/static/app/dash/dash.js [hunk 1]
```diff
@@ -127,5 +148,16 @@ angular.module('dashapp')
...
+            function getParametersByName(name) {
+                var url = window.location.href;
+                name = name.replace(/[\[\]]/g, "\\$&");
+                var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+                    results = regex.exec(url);
+                if (!results || !results[2]) {
+                    return undefined;
+                }
+                return decodeURIComponent(results[2].replace(/\+/g, " ")).split(",");
+            }
```
Comment (ulich) [inline dash-core/src/main/resources/static/app/dash/dash.js]: maybe use `URLSearchParams`? http://stackoverflow.com/a/12151322?

Comment (ulich) [inline dash-core/src/main/resources/static/app/dash/dash.js]: maybe use `URLSearchParams`? http://stackoverflow.com/a/12151322?

Comment (CanardSauvage) [inline dash-core/src/main/resources/static/app/dash/dash.js]: Only works with newer Browsers

**Comment by gaffa** [comment]
What I dont like (even if it may be too late now): it is in dash.js. 
I started to modularize/split the monolith and hope that you guys will continue structuring things.
So maybe this would have been a candidate for a service or something like that? I am not sure.

File: dash-core/src/main/resources/static/app/dash/dash.js (hunk 1 above)
Comment (ulich) [inline dash-core/src/main/resources/static/app/dash/dash.js]: yeah, chrome 49 (http://caniuse.com/#search=URLSearchParams)

if you need to support older browsers, go ahead with this solution

**Comment by CanardSauvage** [comment]
I will update the solution and split it :-)

**Pull request merged**